site/viamguides/src/so101/so101.md

```diff
@@ -0,0 +1,314 @@
+   ![motor setup verification](./assets/setup_app_motor_verify.png)
+1. If verification is successfully, select "Start Calibration ->" to continue to the next workflow. Otherwise, select "<- Previous" to go through the motor setup again.
+   ![motor setup success](./assets/setup_app_motor_verify_success.png)
+
+<!-- ------------------------ -->
+## Calibrate the joints of the arm
+Duration: 10
+
+Now you'll set the home position of the arm and calibrate the range of each joint to ensure accurate movement within Viam.
+If you've already completed this task when assembling the arm, you can configure the arm component to use the generated file in the next step.
+
+### Walk through joint calibration
+
+1. Select "Calibration Only" from the list of workflows (if you've skipped the motor setup).
+1. Review the workspace requirements and safety requirements, then select "Begin Setup Process ->".
+   ![workflow overview step](./assets/setup_app_workflow_overview.png)
+1. Review the additional safety instructions and select "Start Calibration" when ready.
+   ![start calibration](./assets/setup_app_calibration_start.png)
+1. Review the positioning guidelines and arrange the arm into the home position. While holding the arm in place with one hand, select "Set Homeing Position" to record the position.
+   ![save home position](./assets/setup_app_calibration_homing.png)
+   ![arm home position](./assets/arm_home_position.jpeg)
+1. Review the recording process instructions, then select "Start Recording" to stepping through the range of motion for each joint of the arm. Select "Stop Recording" once you've completed this process.
+   ![record joint ranges](./assets/setup_app_calibration_start.png)
+   ![calibration in action](./assets/calibration.gif)
+1. Select "Save Calibration Data" to create the required calibration file on the robot and write the relevant data to each servo motor.
+   ![calibration summary](./assets/setup_app_calibration_summary.png)
+   ![save calibration data](./assets/setup_app_calibration_save.png)
+1. Select "Complete Setup" to review what you've accomplished.
+   ![calibration success](./assets/setup_app_calibration_save_success.png)
+
+You can now close the application and return to your machine configuration to set up the arm and gripper components in Viam.
+
+<!-- ------------------------ -->
+## Configure your so101 arm
+Duration: 10
+
+### Configure the arm component
+1. Click the **+** icon in the left-hand menu and select **Component or service**.
+   ![add component or service](./assets/add_component_service.png)
+1. Search for the `so101:arm` module and select it. 
+    <img alt="find the so101 arm component" src="./assets/search_arm_component.png" width=400 />
+1. Change the name to something descriptive, like `follower-arm`, then click **Create**. This adds the component for working with the first five joints of the robot.
+  <img alt="add the so101 arm component" src="./assets/name_arm_component.png" width=400 />
```

Comment:
I think this line needs to be indented one more segment. If you notice, the rest of the numbering restarts and this particular image is at a top leve (like a full step/instruction) rather than indented to be part of the previous step.